File: mbrs/src/main/resources/static/js/services/CenovnikService.js
//Fri May 11 23:51:24 CEST 2018
//Generisano na osnovu sablona: jsservice.ftl
//Element modela: GremoNaElektriko::Cenovnik

angular.module('gremoNaElektrikoApp.CenovnikService', [])
	.factory('CenovnikService', function($http) {
		var factory = {};
		
		factory.findAll = function() {
			return $http.get('/gremoNaElektriko/cenovnik');
		}
		
		factory.createCenovnik = function(cenovnik ) {
			return $http.post('/gremoNaElektriko/cenovnik', cenovnik);
		}

		factory.deleteCenovnik = function(id) {
			return $http.delete('/gremoNaElektriko/cenovnik/'+id);
		}
		
		factory.updateCenovnik = function(cenovnik) {
			return $http.put('/gremoNaElektriko/cenovnik', cenovnik);
		}
		
		return factory;
	});

File: mbrs/src/main/java/com/ftn/mbrs/service/CenovnikService.java
//Sat May 12 11:32:37 CEST 2018
//Generisano na osnovu sablona: service.ftl
//Element modela: GremoNaElektriko::Cenovnik


package com.ftn.mbrs.service;

import java.util.List;

import com.ftn.mbrs.model.Cenovnik;

public interface CenovnikService {

	public Cenovnik save(Cenovnik cenovnik);
	
	public Cenovnik update(Cenovnik cenovnik);

	public String delete(Long id);
	
	public Cenovnik findOne(Long id);
	
	public List<Cenovnik> findAll();
}
